code/modules/shuttle/shuttle.dm

```diff
@@ -297,6 +299,15 @@
 	initial_engines = count_engines()
 	current_engines = initial_engines
 
+	for (var/turf/T in all_turfs)
+		for(var/obj/effect/shuttle_sound_location/shuttle_loc in T.contents)
+			sound_source = shuttle_loc
```

Comment:
I'm really not a fan of the shuttle sound map marker, making it play from the engines made sense. @optimumtact why'd you have them change it? This method is inherently unfixable/unconstructable in play.

Reply:
there's no way for the effect to get deleted as far as I know.

Reply:
If the turf the marker is on is destroyed it will be left behind. The other half is about shuttle expansion/construction.

Reply:
Go with sound sources on engines. It makes a lot more sense and oranges is being slow about responding. Shuttles are more my thing anyway.

Reply:
Alright, I was waiting for a response to touch this anymore because I can just revert as is to get it working again.
Don't speed merge I'm remembering something being broken but I need to test again.

Reply:
it's the wrong approach because there's no guarantee a shuttle has an engine, the mapper should be in full control of where the sounds are emitted from and how

Reply:
Oh, I thought it was for performance reasons.

Reply:
If there's no engines then a shuttle not making sound makes sense. Just use those engines as the sound source while mapping if for some reason you care about the sound location.

Reply:
Better use engines as sound source. 
If no one on the shuttle you can use random tile or center of the shuttle, 
but if there's no engines then a shuttle not making sound makes sense.

Reply:
You are limiting artistic creativity for absolutely no reason.

The sound source should be a mappable effect.

People may not want shuttle sounds coming from the engine and there is ZERO plausible reason to require it in the code.

Reply:
If you really want it to be possible to designate a sound location on the mapping side then include a helper I suppose but I strongly feel engines should be the default and a map marker be the exception. I don't want to end up in a situation where an aspect of the shuttle cant be constructed in a fashion that makes sense IC.